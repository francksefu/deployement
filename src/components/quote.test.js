import Quotes from "./Quotes";
test("it renders correclty", () => {
    const component = Quotes;
    expect(component).toMatchSnapshot();
});