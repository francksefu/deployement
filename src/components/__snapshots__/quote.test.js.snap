// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`it renders correclty 1`] = `[Function]`;
